codes/python/chapter_sorting/merge_sort.py

```diff
@@ -8,3 +8,41 @@
 sys.path.append(osp.dirname(osp.dirname(osp.abspath(__file__))))
 from include import *
 
+
+"""
+另一种 思路实现归并排序
```

Comment:
Thanks for the hard work!
Considering the correspondence, it is not recommended to create new codes out of the Java version.
Therefore, you'd better remove this function.